todo-app/src/Components/styles/todoItem.css

```diff
@@ -0,0 +1,44 @@
+.todoItem {
+    display: flex;
+    flex-direction: row;
+}
+
+.todoItem > * {
+    text-align: center;
+    flex: 3;
+    padding: 1rem;
+}
+
+.todoItem>*:first-child {
```

Comment:
Careful, this is not a great way to write CSS. Since the `first-child` you are targeting here is a `div` which you are returning in your own JSX, it would be better to just give that `div` a class name.